forem/forem
# Add copy to HTML feature for storybook

## What type of PR is this? (check all applicable)


## Description

This adds a "Copy to HTML" feature for Storybook, meaning that for any crayons component, we can easily generate the HTML to use in a non-Preact area by using the knobs in Storybook then just copy/pasting.

Since we can now toggle settings and directly copy HTML from any storybook story, there's no need for some of the HTML variant stories (since the Preact story now serves both purposes). I've deleted those stories to clean up and keep things streamlined.

NB: The HTML parsing library identified a markup error in our `Article` component related to items inside a `p` (the tl;dr is that a p tag can be implicitly closed, which was occuring in our markup, so the next `/p` didn't relate to anything 🙃  - TIL). I've changed the offending `p` to a `div` as it's more of a "grouping for display" than a paragraph anyway.

## Related Tickets & Documents

N/A

## QA Instructions, Screenshots, Recordings

- Run storybook locally and browse through some stories, adjusting the knobs and viewing the HTML tab
- Run the app locally and check there's no visual change to the home feed cards

### UI accessibility concerns?

Not really, but it's likely that fixing that markup error might have a positive impact on some assistive tech users.

## Added/updated tests?


## [Forem core team only] How will this change be communicated?

_Will this PR introduce a change that impacts Forem members or creators, the
development process, or any of our internal teams? If so, please note how you
will share this change with the people who need to know about it._

      [Admin Guide](https://admin.forem.com/), or
      [Storybook](https://storybook.forem.com/) (for Crayons components)
      [`CHANGELOG.md`](https://github.com/forem/forem/tree/main/CHANGELOG.md)
      or in a [forem.dev](http://forem.dev) post
      replace this line with details on why this change doesn't need to be
      shared_
